### PR TITLE
Fix media bundling

### PR DESCRIPTION
**Pull request by baohx2000**
Media bundle no worky currently

### CODE REVIEW

**Comment by baohx2000** [comment]
:star:

**Comment by baohx2000** [comment]
#162 should be merged before this if possible

File: dev-server.js [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+global.__HOSTNAME__    = process.env.HOST || 'localhost';
+global.__APPTYPE__     = process.env.APP_TYPE || 'shared';
+var __PORT__ = 9000;
 
-process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = '0';
+var getBasePath = function (appType) {
+  switch (appType) {
+    case 'shared':
+    case 'patternlib':
+      __PORT__ = 9000;
```
Comment (zpchavez) [inline dev-server.js]: Seems weird for getBasePath to have the side effect of changing the value or __PORT__.

Comment (baohx2000) [inline dev-server.js]: Just a shortcut to allow you to easily run all of your builds at the same time

File: webpack.config.js [hunk 2]
```diff
@@ -1,43 +1,68 @@
...
 var environment = (process.env.APP_ENV || 'development');
 var npmPath     = path.resolve(__dirname, 'node_modules');
+
+var getBasePath = function (appType) {
+  switch (appType) {
+    case 'shared':
+    case 'patternlib':
+    case 'pattern-library':
+    case 'pl':
+      __PORT__ = 9000;
```
Comment (zpchavez) [inline webpack.config.js]: Same here.

Comment (baohx2000) [inline webpack.config.js]: will rename & rewire so it makes more sense

**Comment by baohx2000** [comment]
@zpchavez , should look better now

**Comment by baohx2000** [comment]
Merged from rebased branch

**Pull request closed without merging**